redis/src/aio/mod.rs

```diff
@@ -72,6 +73,9 @@ pub trait ConnectionLike {
     /// also might be incorrect if the connection like object is not
     /// actually connected.
     fn get_db(&self) -> i64;
+
+    /// Returns `PushManager` of Connection, this method is used to subscribe/unsubscribe from Push types
+    fn get_push_manager(&self) -> PushManager;
```

Comment:
Why do you think this should be part of the trait?

Reply:
I think it could be pulled out of the trait right now, it was used in my prototypes.